readme.md

```diff
@@ -26,6 +47,19 @@ start storybook
 yarn storybook
 ```
 
+start postgres
+```shell
+postgres -D <data_directory>
+```
+
+## Running database migrations using dbmate
+
+Creating migrations: https://github.com/amacneil/dbmate#creating-migrations
+
+Running migrations: https://github.com/amacneil/dbmate#running-migrations
+
+Rolling back migrations: https://github.com/amacneil/dbmate#rolling-back-migrations
```

Comment:
Its okay to link to external sources for more details, but this readme should include the commands needed to create and run migrations. Especially if we're running it in docker.